src/main/java/io/strimzi/kafka/bridge/mqtt/utils/MqttMessageQoS.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.bridge.mqtt.utils;
+
+/**
+ * Represents the MQTT message QoS
+ */
+public enum MqttMessageQoS {
```

Comment:
no need for this class, Netty library has its own you can use `io.netty.handler.codec.mqtt.MqttQoS`